requirements/test.txt

```diff
@@ -122,7 +122,7 @@ pyjwt[crypto]==2.3.0
     #   -r requirements/base.txt
     #   drf-jwt
     #   edx-drf-extensions
-pymongo==4.0.1
+pymongo==3.12.3
```

Comment:
`pymongo` is being downgraded because it has been constrained in `edx/opaque-keys`. See PR https://github.com/openedx/opaque-keys/pull/206 for details.